index.html

```diff
@@ -1,9 +1,92 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>      
+      <div class="upper-div">
+        <div>
+          <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+          <span> Nifty Penguin Magic </span>
+        </div>
+        <nav>
+          <ul>
+            <li><a href="#"> npm Enterprise </a></li>
+            <li><a href="#"> Products </a></li>
+            <li><a href="#"> Solutions </a></li>
+            <li><a href="#"> Resources </a></li>
+            <li><a href="#"> Docs </a></li>
+            <li><a href="#"> Support </a></li>
+          </ul>
+        </nav>
+      </div>
+      
+      <div class="lower-div">
+
+        <div class="left">
+          <img src="images/npm-logo.png" alt="npm logo">
+        </div>
+
+        <div class="center">
+          <img class="search" src="images/magnifying-glass.png" alt="magnifying glass">
+          <form action="" method="get">
+            <input type="text" name="search" id="search" placeholder="Search packages">
+            <button type="submit">Search</button>
+          </form>
+        </div>
+
+        <div class="right">
+          <a href="" class="link-btn join">Join</a>
+          <a href="" class="link-btn">Log In</a>
+        </div>
+
+      </div>
+    </header>
+    
+    <section id="build">
```

Comment:
Aqui poderia ser o Header da sua página, por ser a parte onde contem a idéia principal e os botoes call-to-action

Reply:
Boa, mestre!